src/FSharpLint.Core/Text.resx

```diff
@@ -381,4 +381,7 @@
   <data name="RulesFavourAsKeyword" xml:space="preserve">
     <value>Prefer using the 'as' pattern to match a constant and bind it to a variable.</value>
   </data>
+  <data name="InterpolatedStringWithNoSubstitution" xml:space="preserve">
+    <value>Do not use interpolated string (with $ prefix) or formatting functions (sprintf, failwithf) when not doing interpolation.</value>
```

Comment:
@webwarrior-ws nit, let's change this to "Do not use interpolated string syntax (with $ prefix) or formatting functions (sprintf, failwithf) when not really performing any interpolation."

Reply:
Done.